src/appleseed/renderer/kernel/intersection/intersector.cpp

```diff
@@ -1,563 +1,563 @@
-    // is called on a previously used shading point.
-    debug_poison(shading_point);
-
-    assert(is_normalized(volume_ray.m_dir));
-    assert(volume_ray.get_current_medium() != nullptr);
-
-    // Context.
-    shading_point.m_texture_cache = &m_texture_cache;
-    shading_point.m_scene = &m_trace_context.get_scene();
-
-    // Primary data.
-    shading_point.m_ray = volume_ray;
-    shading_point.m_ray.m_tmax = distance;
-    shading_point.m_primitive_type = ShadingPoint::PrimitiveVolume;
-
-    // Available on-demand results: none.
-    shading_point.m_members = 0;
-}
-
-namespace
-{
-    struct RayCountStatisticsEntry
-      : public Statistics::Entry
-    {
-        uint64  m_ray_count;
-        uint64  m_total_ray_count;
-
-        RayCountStatisticsEntry(
-            const std::string&   name,
-            const uint64         ray_count,
-            const uint64         total_ray_count)
-          : Entry(name)
-          , m_ray_count(ray_count)
-          , m_total_ray_count(total_ray_count)
-        {
-        }
-
-        std::unique_ptr<Entry> clone() const override
-        {
-            return std::unique_ptr<Entry>(new RayCountStatisticsEntry(*this));
-        }
-
-        void merge(const Entry* other) override
-        {
-            const RayCountStatisticsEntry* typed_other =
-                cast<RayCountStatisticsEntry>(other);
-
-            m_ray_count += typed_other->m_ray_count;
-            m_total_ray_count += typed_other->m_total_ray_count;
-        }
-
-        std::string to_string() const override
-        {
-            return pretty_uint(m_ray_count) + " (" + pretty_percent(m_ray_count, m_total_ray_count) + ")";
-        }
-    };
-}
-
-StatisticsVector Intersector::get_statistics() const
-{
-    const uint64 total_ray_count = m_shading_ray_count + m_probe_ray_count;
-
-    Statistics intersection_stats;
-    intersection_stats.insert("total rays", total_ray_count);
-    intersection_stats.insert(
-        std::unique_ptr<RayCountStatisticsEntry>(
-            new RayCountStatisticsEntry(
-                "shading rays",
-                m_shading_ray_count,
-                total_ray_count)));
-    intersection_stats.insert(
-        std::unique_ptr<RayCountStatisticsEntry>(
-            new RayCountStatisticsEntry(
-                "probe rays",
-                m_probe_ray_count,
-                total_ray_count)));
-
-    StatisticsVector vec;
-
-    vec.insert("intersection statistics", intersection_stats);
-
-#ifdef FOUNDATION_BVH_ENABLE_TRAVERSAL_STATS
-    vec.insert(
-        "assembly tree intersection statistics",
-        m_assembly_tree_traversal_stats.get_statistics());
-
-    vec.insert(
-        "triangle trees intersection statistics",
-        m_triangle_tree_traversal_stats.get_statistics());
-#endif
-
-    vec.insert(
-        "triangle tree access cache statistics",
-        make_dual_stage_cache_stats(m_triangle_tree_cache));
-
-    return vec;
-}
-
-}   // namespace renderer
+
```

Comment:
Not sure what happened to this file, did you change the line endings convention maybe?

Reply:
Don't know either what happened there. There are two or three files having this kind of issue, but this is the only one that actually has a change in it (somewhere in these lines there is an `std::` being added). I checked it locally and every IDE told me, only one line has been changed, while `git diff` told me, the whole file has been changed. As I stated before, I had a script add the `std::`s, so I'm not a 100% sure what's going on here.
I will investigate.

Reply:
There was something at the line endings. I have no clue what it was, but it got changed during the file analysis/automatic adaption. The file is back to normal and the respective `std::`s have been added.